source/index/package.json

```diff
@@ -19,7 +19,7 @@
   "devDependencies": {
     "@airswap/order-utils": "0.3.3",
     "@airswap/test-utils": "0.1.1",
-    "@airswap/tokens": "0.1.1",
+    "@airswap/tokens": "0.1.2",
```

Comment:
why does index have tokens within it? can it be removed?

Reply:
i could have missed something, just didn't see it.

Reply:
good spot i dont think it needs it